tests/main/parallel-install-store/task.yaml

```diff
@@ -0,0 +1,28 @@
+summary: Checks for parallel installation of snaps from the store
+
+execute: |
+    #shellcheck source=tests/lib/dirs.sh
+    . "$TESTSLIB"/dirs.sh
+
+    ! snap install test-snapd-tools_foo 2> run.err
+    MATCH 'experimental feature disabled' < run.err
+
+    snap set system experimental.parallel-instances=true
+
+    snap install test-snapd-tools_foo | MATCH '^test-snapd-tools_foo .* installed'
+
+    echo "The snap is listed"
+    snap list | MATCH '^test-snapd-tools_foo '
+    # TODO parallel-install: extend the test to run a snap when the remaining
+    # bits land.
+
+    # TODO parallel-install: extend the test once we can install more than one
+    # instance of a snap from the store
+    ! snap install test-snapd-tools 2> run.err
+    # exact error message:
+    # error: cannot install "test-snapd-tools": cannot refresh, install, or download: The Snap is present
+    #   more than once in the request.
+    MATCH 'cannot install "test-snapd-tools"' < run.err
```

Comment:
Good, this will fail immediately when store starts supporting it, so we don't forget to update the test.